### PR TITLE
[Student][Tests][MBL-14912] | more code coverage

### PR DESCRIPTION
**Pull request by joehoag**
(1) Added a test for ToDo page filtering
(2) Added a basic test for the notification page
(3) Extended the FilesE2ETest to test renaming and deleting of file

### CODE REVIEW

File: apps/student/flank_tablet.yml [hunk 1]
```diff
@@ -1,10 +1,10 @@
 gcloud:
   project: delta-essence-114723
 # Use the next two lines to run locally
-#  app: ./build/outputs/apk/qa/debug/student-qa-debug.apk
-#  test: ./build/outputs/apk/androidTest/qa/debug/student-qa-debug-androidTest.apk
-  app: ./apps/student/build/outputs/apk/qa/debug/student-qa-debug.apk
-  test: ./apps/student/build/outputs/apk/androidTest/qa/debug/student-qa-debug-androidTest.apk
+  app: ./build/outputs/apk/qa/debug/student-qa-debug.apk
```
Comment (Blackchai) [inline apps/student/flank_tablet.yml:4]: Do we want these changes?

File: apps/student/flank_tablet.yml [hunk 2]
```diff
@@ -24,6 +25,6 @@ gcloud:
     orientation: portrait
 
 flank:
-  testShards: 10
+  testShards: 1
```
Comment (Blackchai) [inline apps/student/flank_tablet.yml:28]: This one too, left over from local testing?

File: apps/student/flank_tablet.yml (hunk 1 above)
Comment (joehoag) [reply apps/student/flank_tablet.yml:4]: Oops -- no, I didn't want to commit these changes.  I'll undo them.  Nice catch.

File: apps/student/src/androidTest/java/com/instructure/student/ui/pages/TodoPage.kt [hunk 3]
```diff
@@ -62,6 +72,20 @@ class TodoPage: BasePage(R.id.todoPage) {
...
+        onView(containsTextCaseInsensitive("Favorited Courses")).click()
+        onView(allOf(isAssignableFrom(Button::class.java), containsTextCaseInsensitive("OK"))).click()
+    }
+
+    fun clearFilter() {
+        onView(withId(R.id.clearFilterTextView)).click()
+    }
+
+    fun dismissFilter() {
+
```
Comment (Blackchai) [inline apps/student/src/androidTest/java/com/instructure/student/ui/pages/TodoPage.kt:86]: Do we need this?

**Review by Blackchai — CHANGES_REQUESTED** [review]
Looks good, just might need to revert the flank file changes.

File: apps/student/src/androidTest/java/com/instructure/student/ui/pages/TodoPage.kt (hunk 3 above)
Comment (joehoag) [reply apps/student/src/androidTest/java/com/instructure/student/ui/pages/TodoPage.kt:86]: Doesn't seem like it!  I'll remove it.

**Review by Blackchai — APPROVED** [review]
LGTM!

**Pull request merged**